target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-whw01-v1.dts

```diff
@@ -0,0 +1,211 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019-ap.dk01.1.dtsi"
```

Comment:
I don't think using a device-specific DTSI here, and even across 4019 vs. 4018 platforms, is appropriate.

Reply:
The use of `qcom-ipq4019-ap.dk01.1.dtsi` as a base was done in-line with the patches present in the vendor's GPL package. That said, I've moved this to use `qcom-ipq4019.dtsi` now - which required quite a few additional nodes be added.